packages/e2e-common/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: [
+    'plugin:@monorepo/eslint-plugin-internal/react-typescript-prettier',
+    'plugin:cypress/recommended',
```

Comment:
In future, we can add a new ESLint preset to `packages/eslint-plugin-internal/src/index.js` e.g. `react-typescript-prettier-cypress` that would allow us to define any Cypress ESLint rule overrides if we need them.